src/components/taskDetails/ProgressUpdateCard/latest-progress-update-card.module.scss

```diff
@@ -67,7 +68,9 @@
         .latest-progress-update-card__date-container {
             position: relative;
             display: flex;
-            gap: 5px;
+            align-items: center;
+            justify-content: start;
+            gap: 4px;
```

Comment:
this part logs a warning, please resolve this...
![image](https://github.com/user-attachments/assets/73cbe32b-a216-4169-9ac8-33514b7ae704)